vertical-pod-autoscaler/pkg/admission-controller/resource/pod/patch/resource_updates.go

```diff
@@ -37,13 +41,19 @@ const (
 
 type resourcesUpdatesPatchCalculator struct {
 	recommendationProvider recommendation.Provider
+	maxAllowedCpu          resource.Quantity
 }
 
 // NewResourceUpdatesCalculator returns a calculator for
 // resource update patches.
-func NewResourceUpdatesCalculator(recommendationProvider recommendation.Provider) Calculator {
+func NewResourceUpdatesCalculator(recommendationProvider recommendation.Provider, maxAllowedCpu string) Calculator {
+	var maxAllowedCpuQuantity resource.Quantity
```

Comment:
nit:
```suggestion
	var maxAllowedCPUQuantity resource.Quantity
```

(just because the recommender uses `maxAllowedCPU`, with the upper-case CPU